src/components/Pages/OfficialDocuments/OfficialDocumentEntry.js

```diff
@@ -2,25 +2,47 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl } from 'react-intl';
 import moment from 'moment-timezone';
+import { Link } from 'react-router-dom';
 
 import { officialdocuments as i18n } from 'js/i18n/definitions';
 
-const OfficialDocument = ({ page: {id, date, title, authoringOffice, summary, name, link, pdfSize }, intl }) => {
+const OfficialDocumentEntry = ({ 
+  page: {
+    id,
+    date,
+    title,
+    authoringOffice,
+    summary,
+    name,
+    link,
+    pdfSize,
+    slug,
+    departments
+  },
+  intl
+}) => {
 
   // If the link is a PDF with a pdfSize, then include it.
   const pdfComponent = (!!pdfSize) ?
     <span className="coa-OfficialDocumentPage__pdf-size">(PDF {pdfSize})</span> :
     null
 
+  const entryUrl = (departments, slug) => (
+    (!!departments && !!departments.length) ?
+      `/${departments[0].slug}/${slug}`
+      : '/'
+  )
```

Comment:
Could we return the janis_instance/janis_urls in our graphql query of an OfficialDocumentPage?

Reply:
Not per the schema, but maybe instead I should look into writing a instance/url resolver like our DepartmentResolver? I dont know if it would be any more efficient of a lookup. I'm thinking it over